src/main/webapp/table-manager.js

```diff
@@ -16,6 +17,8 @@ class TableManager {
           this.table.getChart(), 'page', this.onPageChange.bind(this));
     });
 
+    google.visualization.events.addListener(this.table, 'ready', this.makeTableAccessible.bind(this));
+
     this.pageLeft.onclick = () => {
       google.visualization.events.trigger(this.table.getChart(), 'page', {'page': -1});
```

Comment:
maybe replace some of the magic numbers here with final static global vars

Reply:
I've updated the second 'page' to be 'pageDelta' which should make the purpose of the numbers more understandable.